pyproject.toml

```diff
@@ -18,12 +18,12 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.10"
-bask = ">=0.9.3"
+python = ">=3.10,<3.14"
+bask = {git = "https://github.com/AlexisOlson/bayes-skopt.git", branch = "python-3.10-3.13"}
```

Comment:
This needs to be updated before merging.

This branch is now merged into main via
https://github.com/kiudee/bayes-skopt/pull/122